xray/utils/resultstable.go

```diff
@@ -257,7 +259,10 @@ func sortVulnerabilityOrViolationRows(rows []formats.VulnerabilityOrViolationRow
 		if rows[i].SeverityNumValue != rows[j].SeverityNumValue {
 			return rows[i].SeverityNumValue > rows[j].SeverityNumValue
 		}
-		return len(rows[i].FixedVersions) > 0 && len(rows[j].FixedVersions) > 0
+		if len(rows[i].FixedVersions) != len(rows[j].FixedVersions) {
+			return len(rows[i].FixedVersions) > len(rows[j].FixedVersions)
+		}
+		return rows[i].ImpactedDependencyName > rows[j].ImpactedDependencyName
```

Comment:
@sverdlov93 this will effect the audit as well

added sort by package name after fixed version suggestion